### PR TITLE
Fix handling of interactivity

### PR DESCRIPTION
**Pull request by DJMcNab**
We should just block when making SVGs in the headless example, so do that

### CODE REVIEW

File: examples/headless/src/main.rs [hunk 1]
```diff
@@ -104,6 +104,7 @@ async fn render(mut scenes: SceneSet, index: usize, args: &Args) -> Result<()> {
         images: &mut images,
         resolution: None,
         base_color: None,
+        interactive: true,
```
Comment (dfrg) [inline examples/headless/src/main.rs:107]: Should `interactive` be false for headless?

File: examples/with_winit/src/lib.rs [hunk 2]
```diff
@@ -268,6 +268,7 @@ fn run(
                 images: &mut images,
                 resolution: None,
                 base_color: None,
+                interactive: false,
```
Comment (dfrg) [inline examples/with_winit/src/lib.rs:271]: … and true here? That’s what I would expect semantically and seems to align with the actual use of the value but I may be missing something.

Comment (DJMcNab) [reply examples/with_winit/src/lib.rs]: Thanks! :sweat_smile:

**Review by dfrg — APPROVED** [review]
This looks good, thanks!

**Pull request merged**